src/modules/data.js

```diff
@@ -0,0 +1,7 @@
+const getData = async (url) => {
+  let response = await fetch(url);
+  response = await response.json();
+  return response;
```

Comment:
[OPTIONAL] To be able to handle errors successfully, I suggest that you kindly wrap your async functions in a try-catch block.